transform/app/settings.py

```diff
@@ -33,6 +34,12 @@ class GlobalSettings(BaseSettings):
     S3_ENDPOINT: AnyUrl = "https://example.com"
     S3_BUCKET: str = ""
 
+    # - Relations
+    CREATE_RELATIONS: bool = False
+    ORGANIZATION_PROJECT_PATH: str = "input/organizationProject"
```

Comment:
Maybe `ORGANISATION_PROJECT_RELATION_PATH` or `ORG_PROJ_REL_PATH`  - note S in organisation and RELATION. Lets use British version everywhere. Do it also for RESULT_ORGANIZATION_PATH, RESULT_PROJECT_PATH

Reply:
I considered using shorter names, but I wasn't certain. Ultimately, I chose the second option.